Resources/public/js/views/services/ez-contentcreateviewservice.js

```diff
@@ -56,9 +56,15 @@ YUI.add('ez-contentcreateviewservice', function (Y) {
             var content = new Y.eZ.Content(),
                 version = new Y.eZ.Version(),
                 type = this.get('contentType'),
-                defaultFields = {};
+                defaultFields = {},
+                contentTypeNames,
+                contentTypeName;
 
-            content.set('name', 'New "' + this.get('contentType').get('names')['eng-GB'] + '"');
+            contentTypeNames = this.get('contentType').get('names');
+            contentTypeName = contentTypeNames[this.get('app').get('defaultLanguageCode')]
```

Comment:
@andrerom  one problem at a time please :) You are right but keep in mind that:

* this `contentTypeName` is only used as a title in the form so if we pick the wrong content type name ok that's a bug but a minor one in this case and a different one than https://jira.ez.no/browse/EZP-25678
* having a prioritized language list basically means we are on a siteaccess but until the PlatformUI siteaccess handling, this is a very unclear until the corresponding EPIC correctly specified.
* we have the same problem in a lots of places, so the solution is not to fix it there and that will be a quite large fix